VisDataProvider.cpp

```diff
@@ -179,6 +179,17 @@ int VisDataProvider::pull() {
     return 0;
 }
 
+bool VisDataProvider::waitConnection(int s) const {
+    ALOGV("waitConnectionMs");
+    int minSleepTime = std::min(s, 1);
```

Comment:
As I understand this line is equal to "int minSleepTime = 1;".
std::min will return 1 always except "s=0", but this case I will review bellow.